gemcalibration/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Makefile for gemcalibration package
+#
+
+Project=cmsgemos
+ShortProject=gem
+Package=gemcalibration
+LongPackage=gemcalibration
+
+ShortPackage=gemcal
```

Comment:
`ShortPackage` (and namespaces)` should be `calib`, i.e., 
* `gem::calib::blah`
* `include "gem/calib/file.h"`
* dir structure `include/gem/calib`